.expeditor/habitat-test.pipeline.yml

```diff
@@ -3,7 +3,18 @@ steps:
 
 - label: ":linux: Validate Habitat Builds of Chef Infra"
   commands:
-    - hab pkg install chef/chef-infra-client
+    - hab pkg install "$EXPEDITOR_PKG_IDENTS_CHEFINFRACLIENTX86_64LINUX"
```

Comment:
😑 Oh, environment variables that don't support hyphens. I will start reading this as Chefin' Fracli Ent.